internal/db/migrations-tern/008_recreate_table_blueprints.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE blueprints;
+
+CREATE TABLE IF NOT EXISTS blueprints
+(
+    id uuid PRIMARY KEY,
+    org_id varchar NOT NULL CHECK (coalesce(trim(org_id), '') != ''),
+    account_number varchar NOT NULL CHECK (coalesce(trim(account_number), '') != ''),
+    name VARCHAR(100) NOT NULL,
+    description VARCHAR(250),
+    created_at TIMESTAMP NOT NULL DEFAULT current_timestamp
+);
+
+CREATE UNIQUE INDEX ON blueprints(org_id);
```

Comment:
```suggestion
CREATE INDEX ON blueprints(org_id);
```

I have missed this one, this index must be not unique of course.